server/schema/Resolvers.ts

```diff
@@ -151,28 +150,16 @@ const resolvers: IResolvers = {
                         apiKey,
                     });
 
-                    // save project and store new ID
-                    const resultID: string | undefined = await newProject
+                    // save project
+                    return newProject
                         .save()
-                        // eslint-disable-next-line no-underscore-dangle
-                        .then((res: Project): string | undefined => res._id?.toString())
                         .catch(
                             (err: Error): Error =>
                                 new Error(`Saving project/receiving new ID from DB failed: ${err}`)
-                        );
-
-                    /* updates project in DB to include API endpoint for logger
-                     * endpoint (sans API Key) -> host.com/:userID?project=[projectID]
-                     * user enters this (along with API key in header) into their project's
-                     * src code with gate-logger library installed
-                     */
-                    return ProjectDB.findByIdAndUpdate(
-                        resultID,
-                        { endpoint: `/log?project=${resultID}` },
-                        { new: true }
-                    ).then((data: Project): Project => data);
```

Comment:
This functionality is no longer required as we do not need an endpoint in the DB (and have already removed it from the DB model)